app/locale/fr.coffee

```diff
@@ -1517,7 +1520,7 @@ module.exports = nativeDescription: "français", englishDescription: "French", t
 #    students_not_assigned: "Students who have not been assigned {{courseName}}"
     course_overview: "Vue d'ensemble de cours"
     copy_class_code: "Copier le code de la classe"
-#    class_code_blurb: "Les étudiants peuvent rejoindre leur classe en utilisant ce code propre à la classe. Pas d'adresse mél est requise quand on crée un compte d'étudiant avec ce code propre à sa classe."
```

Comment:
Example of translation we would be losing because we had missed uncommenting it.